src/connection/server_support.c

```diff
@@ -1370,26 +1423,9 @@ css_init (THREAD_ENTRY * thread_p, const char *server_name, int port_id)
       css_Master_server_name = strdup (server_name);
       css_Master_port_id = port_id;
       css_Pipe_to_master = conn->fd;
```

Comment:
How about putting them into the above `else` clause and setting them `css_register_ha_server()` in the case of HA? I know they are the same value here and there finally, but I think it's better if it's set by the same source at the same time it's used, which is more safe and open to change.
```suggestion
      css_Master_server_name = strdup (server_name);
      css_Master_port_id = port_id;
      css_Pipe_to_master = conn->fd;
```